sw/device/tests/alert_handler_reverse_ping_in_deep_sleep_test.c

```diff
@@ -175,7 +202,18 @@ static void check_local_alerts(void) {
  * line to the CPU, which results in a call to this OTTF ISR. This ISR
  * overrides the default OTTF implementation.
  */
-void ottf_external_isr(void) { interrupt_serviced = true; }
+void ottf_external_isr(void) {
+  // Specifically assert that no alerts were triggered
+  check_local_alerts();
+  dif_rv_plic_irq_id_t irq_id;
+  CHECK_DIF_OK(dif_rv_plic_irq_claim(&plic, kPlicTarget, &irq_id));
```

Comment:
This is the important part, if we don't do this we wont exit the interrupt handler and locally I'm seeing it hang here.

The rest is just debugging info to help show why the alert has fired in my local case.

Reply:
As written, that's not precisely what I'd expect to see. Without the claim, the PLIC will never stop asserting the external interrupt signal, and the CPU will spin in and out of the ISR. The CPU does exit the ISR, but it immediately enters it again due to the still-asserted interrupt line.

With only the claim, we don't actually finish the handling, though. The interrupt has been claimed but not completed, so this artificially prevents other interrupts from triggering. In the general case, this ISR has more work to do. However, deferring the completion _might_ be okay, depending on how this is used (I haven't done any analysis on that yet :) )